jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -633,10 +633,6 @@ YUI.add('juju-env-api', function(Y) {
       @return {undefined} Sends a message to the server only.
     */
     loginWithMacaroon: function(bakery, callback) {
-      if (this.pendingLoginResponse) {
```

Comment:
These values are relied on in many places, why was it removed? Have you resolved the pending login race condition here somehow?